### PR TITLE
Map back end

### PR DESCRIPTION
**Pull request by Franck59840**

### CODE REVIEW

**Review by NacymeG — DISMISSED** [review]
Ok for me !

File: routes/localisation.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const router = express.Router();
+const { db } = require('../db-config');
+
+router.get('/', async (req, res) => {
+  try {
+    const [location] = await db.query(`
+    SELECT id, type, name, streetNumber, street, postalCode, city, latitude, longitude, phone
+    FROM location
+  `);
+    console.log(location);
```
Comment (Hacene010) [inline routes/localisation.js]: console.log ???

File: routes/localisation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    res.json(location);
```
Comment (Hacene010) [inline routes/localisation.js]: Forgot the status of the response.

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some evils

**Review by Hacene010 — APPROVED** [review]

**Review by NacymeG — APPROVED** [review]

**Pull request merged**